src/modules/commander/commander_params.c

```diff
@@ -344,9 +344,12 @@ PARAM_DEFINE_FLOAT(COM_OF_LOSS_T, 0.0f);
  * The offboard loss failsafe will only be entered after a timeout,
  * set by COM_OF_LOSS_T in seconds.
  *
- * @value 0 Land mode
- * @value 1 Hold mode
- * @value 2 Return mode
+ * @value -1 Disabled
+ * @value  0 Land mode
+ * @value  1 Hold mode
+ * @value  2 Return mode
```

Comment:
Why no "Land mode"?

Reply:
I do not understand, land is value 0. Can you elaborate?

Reply:
Ah :man_facepalming:, nevermind.

Reply:
No problem :-D